src/App.js

```diff
@@ -1,8 +1,10 @@
 import {useState} from 'react';
+import { Routes, Route} from 'react-router-dom';
 import styled from 'styled-components';
 
-import ChairCard from './component/ChairCard/ChairCard';
+import ChairMapping from './component/ChairMapping/ChairMapping';
 import data from './data.json';
+import Home from './pages/Home';
 
 export default function App() {
   const chairsData = data;
```

Comment:
You can erase this line if you import your data in line 6 directly as chairsData:

`import chairsData from './data.json';`